content/en/quality_gates/explorer/facets.md

```diff
@@ -0,0 +1,124 @@
+
+**Note:** Although facets are not required for filtering on tags, defining facets for tags that you often use during investigations can help reduce your time to resolution.
+
+### Quantitative measures
+
+Use quantitative measures to:
+
+- **Aggregate** values from multiple quality gates.
+- **Range filter** your quality gates.
+- **Sort** your quality gates against a value.
+
+#### Types
+
+Measures support long integers and double values.
+
+#### Units
+
+Measures support units (**time** in seconds or **size** in bytes) for handling of orders of magnitude at query time and display time. The unit is a property of the measure itself, not of the field.
+
+For example, consider a `duration` measure in nanoseconds. Suppose test runs from `service:A` have `duration:10000000`, meaning `10 milliseconds`. Supposed test runs from `service:B` have `duration:5000000`, meaning `5 milliseconds`. Use `duration:>2ms` to consistently query test run tags from both services at once. For more information about search queries, see [Search Syntax][6].
+
+## Facet panel
+
+The search bar provides a comprehensive set of interactions to filter and group your data. However, for many cases, the facet panel is a straightforward way to navigate into your data. Open a facet to see a summary of its content for the scope of the current query.
+
+The search bar and URL automatically reflect your selections from the facet panel.
+
+- **Facets (qualitative)** show a list of unique values, and a count of quality gates that match each facet.
+- **Measures (quantitative)** have a slider ranging between minimum and maximum values. Use the slider, or input numerical values, to scope the search query to different bounds.
+
+### Grouping facets
+
+Facets are grouped into meaningful themes in the facet list. Assigning or reassigning a group for a facet affects only the facet list, and has no impact on search or analytics.
+
+### Filtering facets
+
+Use the search facets field on the facet panel to filter the facet list and navigate to a particular facet. The search uses the facet display name and field name to scope results.
+
+## Creating facets
+
+Creating a facet on a test run or pipeline execution attribute or tag is not a mandatory step to search for quality gates. Facets are useful if you wish to add a meaningful description to a specific test run or pipeline execution attribute, or if you want the attribute values to appear on the Facets list.
+
+### Creating facets from the facet list
+
+You can create a facet directly from the facet panel by clicking **+ Add**.
+
+{{< img src="quality_gates/explorer/facets.png" alt="Add a facet from the facet side panel" style="width:30%;">}}
+
+Define the underlying field (key) name for this facet:
+
+- Use tag key name for infrastructure tags.
```

Comment:
Is infrastructure tags relevant here? or can we remove it?

Reply:
```suggestion
```